v8_c_bridge.cc

```diff
@@ -421,6 +421,39 @@ ValueErrorPair v8_Value_Call(ContextPtr ctxptr,
   };
 }
 
+ValueErrorPair v8_Value_New(ContextPtr ctxptr,
+                            PersistentValuePtr funcptr,
+                            int argc, PersistentValuePtr* argvptr) {
+  VALUE_SCOPE(ctxptr);
+
+  v8::TryCatch try_catch;
+  try_catch.SetVerbose(false);
+
+  v8::Local<v8::Value> func_val = static_cast<Value*>(funcptr)->Get(isolate);
+  if (!func_val->IsFunction()) {
+    return (ValueErrorPair){nullptr, DupString("Not a function")};
+  }
+  v8::Local<v8::Function> func = v8::Local<v8::Function>::Cast(func_val);
+
+  v8::Local<v8::Value>* argv = new v8::Local<v8::Value>[argc];
+  for (int i = 0; i < argc; i++) {
+    argv[i] = static_cast<Value*>(argvptr[i])->Get(isolate);
```

Comment:
I see that this is what I did in v8_Value_Call -- but out of curiosity, can you use argvptr directly?  It's a list of values, essentially, but in the wrapper classes.  My build is broken at the moment so I can't test it out myself.